gnomad_qc/v4/resources/meta.py

```diff
@@ -29,23 +30,31 @@ def meta_tsv_path(version: str = CURRENT_VERSION, data_type: str = "exomes") ->
     )
 
 
-def meta(data_type: str = "exomes") -> VersionedTableResource:
+def meta(
+    version: str = CURRENT_VERSION, data_type: str = "exomes", overwrite: bool = False
```

Comment:
this is contradictory to my other comments, but I don't think this overwrite needs to be an argument -- you wouldn't ever want to set this to True, since this is only used to check if the meta already exists. it also cleans up calls to this function; e.g.
```
    ht.write(
        v4_meta(data_type="genomes", overwrite=args.overwrite).path,
        overwrite=args.overwrite,
    )
```
could be reverted to
```
    ht.write(v4_meta(data_type="genomes").path, overwrite=args.overwrite)
```

Reply:
I think because I need to test the script, `check_resource_existence` here will throw an error when it exists. I will revert it.